libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -691,26 +691,7 @@ bool ContractCompiler::visit(InlineAssembly const& _inlineAssembly)
 		{
 			int const depositBefore = _assembly.stackHeight();
 			solAssert(!!decl->type(), "Type of declaration required but not yet determined.");
-			if (FunctionDefinition const* functionDef = dynamic_cast<FunctionDefinition const*>(decl))
```

Comment:
When was this last used? When did we export functions to assembly and when did we stop?

Reply:
some time ago.

Reply:
Reassuring.

Reply:
Would really prefer to have an idea when we decided to do allow/disallow that.